docs/introduction.rst

```diff
@@ -1,142 +1,132 @@
 Introduction to Flatpak
 =======================
 
-Flatpak is a framework for distributing desktop applications across various Linux
-distributions. It has been created by developers who have a long history of working on the
-Linux desktop, and is run as an independent open source project.
+Flatpak is a framework for distributing desktop applications across various
+Linux distributions. It was created by developers with a long history of
+working on the Linux desktop and is run as an independent open source project.
 
 Terminology
 -----------
 
 - Flatpak: a system for building, distributing, and running sandboxed desktop applications on Linux.
-- Flatpak application: these are the applications the user installs via the ``flatpak`` command or via a
-  different UI like GNOME Software or KDE Discover.
-- Runtime: also called platforms, these are integrated platforms to provide basic utilities needed for a
-  Flatpak application to work.
-- BaseApp: these are integrated platforms for frameworks like Electron.
-- Flatpak bundle: a specific single-file export format which contains a Flatpak app or runtime.
+- Flatpak application: an application installed via the ``flatpak`` command or through a
+  graphical interface, such as GNOME Software or KDE Discover.
+- Runtime: also called platform, an integrated environment providing basic
+  utilities needed for a Flatpak application to work.
+- BaseApp: an integrated set of additional libraries and frameworks, such as Electron, for Flatpak
+  applications that need more than just the basic runtime.
+- Flatpak bundle: a single-file export format containing a Flatpak appplication or runtime.
 
 Target audience
 ---------------
 
-Flatpak can be used by all kinds of desktop applications, and aims to be
-as agnostic as possible regarding how applications are built. There are no
+Flatpak can be used by all kinds of desktop applications and aims to be as
+agnostic as possible in terms of how applications are built. There are no
 requirements regarding which programming languages, build tools, toolkits
 or frameworks can be used.
 
 While Flatpak only runs on Linux, it can be used by applications that target
-other operating systems, as well as those that are Linux-specific. Applications
+other operating systems as well as those that are Linux-specific. Applications
 can be open source or proprietary (although some distribution services, like
 `Flathub <https://flathub.org/>`_, can have restrictions in this respect).
 
-The only technical requirements made by Flatpak are that applications follow a
-small number of Freedesktop standards, in order to enable desktop integration
+Flatpak's only technical requirements are that applications follow a
```

Comment:
ran it through a grammar checker and it complains that plural requirements does not align with `a few`.

Reply:
I don't think `a few` here is grammatically related to `requirements` but will revert it as this is an unnecessary change.